cloudbuild/nightly/scripts/python-scripts/create_unbounded_sink_table.py

```diff
@@ -0,0 +1,72 @@
+import argparse
```

Comment:
Let's add a few lines describing the test here, that we have an initial source and we create another folder copy it and so on... 2-3 lines only.

Reply:
Also mention that this file is for internal, unbounded read-write tests only.